MapboxAndroidDemo/src/main/res/layout/activity_dds_symbol_collision.xml

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
two textviews in this layout use the same label `icon_overlap_explanation`, while one should reference `icon_placement_explanation` instead.

Reply:
🔍fixed 🙇